schema_based_on_diagram.sql.sql

```diff
@@ -1 +1,49 @@
-/**All db schema*/
\ No newline at end of file
+CREATE TABLE patients (
+    id INT PRIMARY KEY,
+    name VARCHAR(250) NOT NULL,
+    date_of_birth DATE
+);
+
+CREATE TABLE history_treatment_id (
+  medical_history_fk INT PRIMARY KEY,
+  treatment_fk INT PRIMARY KEY
+);
+
+CREATE TABLE medical_histories (
+    id INT PRIMARY KEY,
+    FOREIGN KEY (id) REFERENCES history_treatment_id(medical_history_fk),
```

Comment:
- [ ] When the former reviewer said you should create foreign keys, he meant you should create a foreign key for the `medical_histories` table in the `history_treatment_id` table. Kindly remove this foreign key here. 👍